pkg/asset/installconfig/installconfig.go

```diff
@@ -133,6 +135,30 @@ func (a *InstallConfig) finishGCP() error {
 		}
 		a.Config.Platform.GCP.Endpoint.ClusterUseOnly = &defaultClusterUseOnly
 	}
+
+	project := a.Config.GCP.ProjectID
+	if a.Config.Platform.GCP.NetworkProjectID != "" {
+		project = a.Config.GCP.NetworkProjectID
+	}
+
+	if a.Config.GCP.FirewallRulesManagement == "" {
```

Comment:
The default value of `platform.gcp.firewallRulesManagement` is `Managed`, is it the case? 

```
$ openshift-install explain installconfig.platform.gcp.firewallRulesManagement
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  Default: "Managed"
  Valid Values: "Managed","Unmanaged"
  FirewallRulesManagement specifies the management policy for the cluster. Managed indicates that
the firewall rules will be created and destroyed by the cluster. Unmanaged indicates that the
user should create and destroy the firewall rules.

Reply:
Ah, right. We need to update the go doc for the field remove `// +default="Managed"` and add a comment to mention that the installer set the default based on whether a set of required permissions are available.

https://github.com/openshift/installer/blob/d921ca51ac605ef81ef83b3ba0583e8fac7ae382/pkg/types/gcp/platform.go#L124-L127

Reply:
sounds good. Changes made